### ISSUE TITLE
proselyte of the sakura clan is missing

### ISSUE THREAD

**Issue by sanymany** [ValveSoftware/Dota2-Gameplay#20148]
### Description

when i got this skin i clicked unbundle but my skin was recycled and i dont know why. I CLICKED UNBUNDLE GIVE ME MY SKING BACK

### Example Match ID (and possibly Timestamp)

22:30 

### Screenshots

![Uploading Снимок экрана 2024-07-17 223540.png…]()

**Comment by josevitorsargentini** [comment]
Yeah. I bought all 13 caches and all my sets got recycled. They ain't gonna do shit about it.
